src/components/main-page/EventDetailsExtended/EventDetailsExtended.tsx

```diff
@@ -9,26 +9,96 @@ export function EventDetailsExtended() {
+        <strong>mostem łączącym uczestników i firmy partnerskie</strong> poprzez
+        inspirujące rozmowy, czy możliwość nawiązania cennych kontaktów.
+      </Text>
+      <Text className={styles.text}>
+        Podczas konferencji, uczestnicy będą mieli okazję uczestniczyć w{" "}
+        <strong>
+          prelekcjach prowadzonych przez doświadczonych ekspertów branżowych
+        </strong>
+        . Poruszane tematy obejmują różnorodne aspekty kształtowania kariery i
+        samorozwoju.
+      </Text>
+
+      <Text className={styles.text}>
+        AKAI Camp to także doskonała okazja do nawiązania relacji z
+        przedstawicielami firm partnerskich. Będziesz miał możliwość zadawania
+        pytań, dowiedzenia się więcej o ich oczekiwaniach wobec potencjalnych
+        pracowników oraz poznania aktualnych trendów w branży. To wszystko
+        sprawia, że AKAI Camp to nie tylko miejsce edukacji, ale także
+        inspiracji i <strong>networkingu</strong>!
+      </Text>
+
+      <Text className={styles.text}>
+        Nie możemy się doczekać, aby Cię przywitać na AKAI Camp i razem
+        eksplorować możliwości poszerzania naszych ścieżek zawodowych, czerpiąc
+        z bogactwa wiedzy i doświadczenia naszych wyjątkowych prelegentów oraz
+        partnerów. Zapisy ruszają wkrótce, więc obserwuj nasze social media i
+        stań się częścią tej wyjątkowej przygody!
+      </Text>
 
       <Heading size="xl" as="h3" className={styles.subtitle}>
-        Tytuł
+        Dla kogo?
       </Heading>
 
-      <Text className={styles.text}>Tekst</Text>
-      <Text className={styles.text}>Tekst</Text>
+      <Text className={styles.text}>
+        Jesteś studentem? Licealistą? A może po prostu pragniesz poszerzać
+        horyzonty i wzbogacić swoje CV? AKAI Camp jest dedykowany właśnie dla
+        Ciebie!
+      </Text>
 
       <Heading size="xl" as="h3" className={styles.subtitle}>
-        Podtytuł
+        Panele dyskusyjne
       </Heading>
 
+      <Text className={styles.text}>
+        Sercem wydarzenia są dwa równoległe panele dyskusyjne, które będą trwały
+        przez cały dzień:
+      </Text>
+
       <div className={styles.categoriesContainer}>
-        <Category icon={lightBulb} title={"Tytuł"} text={"Tekst"} />
-        <Category icon={cog} title={"Tytuł"} text={"Tekst"} />
-        <Category icon={hammerWrench} title={"Tytuł"} text={"Tekst"} />
+        <Category
+          icon={lightBulb}
+          title={"Meet my career"}
+          text={
+            "Ten panel skupi się na kluczowych umiejętnościach niezbędnych do osiągnięcia zawodowych celów. Przedstawione zostaną ważne błędy, których warto unikać, by osiągnąć sukces jako aspirujący profesjonalista. Czeka na Was też wartościowa analiza różnych ścieżek zawodowych, podczas której zaprezentowane zostaną ich zalety i wyzwania. Będziemy omawiać, na co szczególnie zwracać uwagę, aby zdobyć przewagę na dynamicznym rynku pracy."
```

Comment:
do pooprawy ten opis - trzerba zaznaczyć że jest to panel poświęcony też stricte umiejętnościom technicznym